### PR TITLE
docs: Fix typos and grammatical errors

### PR DESCRIPTION
**Pull request by vinayak42**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Fixes typos and grammatical errors

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ You can assign up to 10 categories per message:
 ```
 
 <a name="schedulling-parameters"></a>
```
Comment (hugovk) [inline USAGE.md]: How about also fixing the typo in the link here and in the table of contents?

```suggestion
<a name="scheduling-parameters"></a>
```

File: USAGE.md [hunk 2]
```diff
@@ -213,7 +213,7 @@ The format of the SMTP API section tag has the form:
 ```
 
 <a name="subtitution-tags"></a>
```
Comment (hugovk) [inline USAGE.md]: How about also fixing the typo in the link here and in the table of contents?
```suggestion
<a name="substitution-tags"></a>
```

**Comment by vinayak42** [comment]
@hugovk Applied changes as suggested

**Comment by thinkingserious** [comment]
Hello @vinayak42,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**